src/api/controllers/commentController.js

```diff
@@ -126,21 +126,24 @@ class CommentController {
   }
 
   /**
-   * A method to update a single comment
-   *
+   * @description this method stores the updated comment
+   * that  is passed by the comparison middleware.
+   * @success it will respond with message asuccess message to
+   *  inform the user of successful execution of the task.
    * @author Prémices
+   * @modified Daniel Charles Mwangila
    * @static
    * @param {object} req - this is the request object
    * @param {object} res - this is the response object
    * @returns {object} - this function returns an object
    * @memberof RatingsController
    */
   static async updateComment(req, res) {
-    // Initializing variables
     const { slug, id } = req.params;
+    const { body, iteration, isEdited } = req.newComment;
 
     await Comment.update({
-      body: req.body.body
+      body, iteration, isEdited
```

Comment:
@truestbyheart this one too

Reply:
I want to simplify the work reduce too much ``if``s by simply identifying the comment is an update...rather than to start and write code to check if the iteration is greater then one